src/main.cpp

```diff
@@ -89,6 +89,7 @@ bool fSpentIndex = false;       // insightexplorer
 bool fTimestampIndex = false;   // insightexplorer
 bool fHavePruned = false;
 bool fPruneMode = false;
+int64_t nPreferredTxVersion = DEFAULT_PREFERRED_TX_VERSION;
```

Comment:
Transaction versionsn are only 32 bits.

Reply:
`GetArg` returns 64 bit values though. I guess I have to manually check the range of the value returned by `GetArg` and fail if it's out of the allowed range.